Feature bilinear

pnuu opened the pull request:
Added access to pyresample.bilinear as a new resampling method. Includes caching for the resampling coefficients, and they can be saved to disk and re-used (with precompute=True argument) in the same way as with nearest neighbour resampling.

pnuu commented:
Removed Travis tests for Python 2.6.

None of the available Pillow versions from pypi work. Pillow 4.0.0 drops support for Python 2.6, 3.4.2 isn't able to create empty 0-by-0 images and 1.7.8 (from 2012) requires an integer argument for saving PNG images.

mraspaud commented on mpop/projector.py:
```diff
@@ -227,6 +227,20 @@ def __init__(self, in_area, out_area,
                 self._cache['ewa_rows'] = rows
 
+            elif self.mode == "bilinear":
+                from pyresample.bilinear import get_bil_info
```
Can this be moved to the top of the file ?

mraspaud commented on mpop/projector.py:
```diff
@@ -282,4 +296,21 @@ def project_array(self, data):
                                            rows_per_scan=rows_per_scan)
 
+        elif self.mode == "bilinear":
+            from pyresample.bilinear import get_sample_from_bil_info
```
Move to file top ?

pnuu commented:
I updated the docstring and added a test that the bilinear cache is generated when the Projector class is instantiated (and eventually got the tests to work..).
